src/Synergism.ts

```diff
@@ -1757,7 +1757,7 @@ export const format = (
         return isNaN(input as number) ? '0 [NaN]' : '0 [und.]';
     } else if ( // this case handles numbers less than 1e-6 and greater than 0
         typeof input === 'number' &&
-        input < 1e-12 && // arbitrary number, can be changed
+        input < (!fractional ? 1e-3 : 1e-15) && // arbitrary number, don't change 1e-3
```

Comment:
?

Reply:
this is a live change now, was awwhy's bug fix for global speed not showing negative exponents. without it I had the same bug in the history tab